lib/PEXv2.ts

```diff
@@ -115,7 +118,7 @@ export class PEXv2 {
    */
   public selectFrom(
     presentationDefinition: PresentationDefinitionV2,
-    verifiableCredentials: OriginalVerifiableCredential[],
+    verifiableCredentials: (IVerifiableCredential | JwtDecodedVerifiableCredential | CompactJWT)[],
```

Comment:
This is literally what OriginalVerifiableCredential is

Reply:
fixed